### DISCUSSION TITLE
What if your data spans the entire world?

### DISCUSSION THREAD

**Discussion by leoohyama** [sdmTMB/sdmTMB#134] [Q&A]
Hi all,

Working with global data and was wondering what projection should be used for my data? Using the function `sdmTMB::add_utm_columns`  gives me UTM zone 19N; CRS = 32619 which is not ideal and cuts off most of my study's space. I guess UTM zones are not great for the global scale? 

Any thoughts would be appreciated!

**Comment by seananderson** [comment]
For non-global data spanning well beyond one UTM zone boundary (e.g., across a continent), a projection such as the Albers might make sense. We do that in the snowy owl appendix of the [sdmTMB preprint](https://doi.org/10.1101/2022.03.24.485545) (specific code [here](https://github.com/seananderson/sdmTMB-paper/blob/f363ccec5d493a6b613bc5d074dd12ab17c0b5f1/doc/appendix-snow-sdmTMB.Rmd#L67-L72)). But for global data, you'd probably want to use a spherical mesh so that correlation can wrap around. INLA can build those meshes if you set the `globe` argument to a non-null value where the number corresponds to the 'subdivision resolution'. E.g. `mesh <- INLA::inla.mesh.create(globe = m)`. However, I have no idea if that would just work within sdmTMB, if we'd have to tweak something simple, or if that would be a major undertaking. I don't have a simple example dataset to try with, but if you can supply one I'd be happy to look into this.

**Reply by leoohyama** [reply]
Thanks for the quick reply! 

I can provide a quick and simple dataset that comes in a rds format. We utilized hexagonal bins for our data but used centroids of these bins as spatial coordinates. So the attached data (txt file) is point data (WGS84) without any projection and needs to be converted with sf. The response variables of interest are `avg_size` and `prop_om` (proportion of omnivores). The predictors include MAT, MAP (temp., precipitation) and ATR (annual temp. range). I've mostly been experimenting with single predictor models so far to make sure models converge. 


[example_df.txt](https://github.com/pbs-assess/sdmTMB/files/9895622/example_df.txt)

**Reply by leoohyama** [reply]
Hi @seananderson , just following up on this issue and wondering if you were able to take a look at this!
